scripts/setup.sh

```diff
@@ -6,17 +6,28 @@
 #
 # Commands that may fail have "|| return" to avoid continuing or interfering with terminal.
```

Comment:
Because most of the `|| return` have been removed, perhaps we should remove the comment [L7](https://github.com/artsy/metaphysics/blob/3c79914b378e75b040a8074919070c6148835a03/scripts/setup.sh#L7) and update the comment on [L4](https://github.com/artsy/metaphysics/blob/3c79914b378e75b040a8074919070c6148835a03/scripts/setup.sh#L4) to match the README change:

```
Run like:
  ./scripts/setup.sh
```